src/core/AutoRest.Extensions/ParameterGroupExtensionHelper.cs

```diff
@@ -35,6 +35,7 @@ private static Property CreateParameterGroupProperty(Parameter parameter)
                 //Constraints = parameter.Constraints, Omit these since we don't want to perform parameter validation
                 Documentation = parameter.Documentation,
                 ModelType = parameter.ModelType,
+                RealPath = new string[] { },
```

Comment:
fixes ArgNullEx when using both `x-ms-parameter-grouping` and media type `application/xml` in the same operation (#2236)